### PR TITLE
Update schema

### PR DESCRIPTION
**Pull request by lbraun**
Hey @alaacs,

Here are the changes I made to the schema tonight. Can you just look them over to make sure I didn't do anything nonsensical? Gracias!

### CODE REVIEW

File: database/schema.sql [hunk 1]
```diff
@@ -2,6 +2,11 @@ CREATE DATABASE csr_lookup;
 
 \connect csr_lookup;
```
Comment (alaacs) [inline database/schema.sql]: We can add this line before creating the table so it gets dropped before creating.
DROP TABLE IF EXISTS table_name;

Comment (lbraun) [reply database/schema.sql]: Good idea, I just added it. Only have to make sure we don't run it by accident 😆

Comment (alaacs) [reply database/schema.sql]: No bro, I mean for each table, do you want to drop the whole database and recreate it?
Sounds unusual to me but why not!
But you will need to edit the line to:
DROP **DATABASE**  IF EXISTS  csr_lookup

Comment (lbraun) [reply database/schema.sql]: Oh man :man_facepalming: now it should actually work. I think it makes sense to drop the whole db to make sure you're really starting with a clean slate.

Comment (alaacs) [reply database/schema.sql]: Awesome!

**Pull request merged**